cron/internal/data/projects.csv

```diff
@@ -1243372,8 +1243372,8 @@ github.com/zowe/explorer-uss,num_dependents_deps.dev:0
 github.com/zowe/file-transfer-app,num_dependents_deps.dev:0
 github.com/zowe/imperative,
 github.com/zowe/jobs,num_dependents_deps.dev:0
-github.com/zowe/launcher,num_dependents_deps.dev:0
```

Comment:
What was the reason for these changes?

Reply:
When running `make all` the build process automatically made these changes in `projects.csv`. I will remove them.